### ISSUE TITLE
nf=10x-pathseq  give viral content in each single cell or each sample

### ISSUE THREAD

**Issue by liangdp1984** [redst4r/nf-10x-pathseq#1]
nice tool!  Due to low coverage of 10x scRNA, is it possible to measure viral content in each cell? it seems that nf-10x-pathseq output viral content in each sample!

**Comment by redst4r** [comment]
yeah, might be hard to do it on a cell by cell basis. It's technically possible (via the CB/UMI of the reads), but probably not very reliable (just because you dont detect a viral read for a given cell doesn't mean the cell doesn't have some viral RNA). On a sample level it's much more robust.
